src/rust/inetstack/protocols/tcp/tests/setup.rs

```diff
@@ -301,10 +300,10 @@ fn test_refuse_connection_missing_syn() -> Result<()> {
     advance_clock(Some(&mut server), Some(&mut client), &mut now);
 
     // Server: SYN_RCVD state at T(2).
-    match server.receive(buf) {
-        Err(error) if error.errno == EBADMSG => Ok(()),
-        _ => anyhow::bail!("server receive should have returned an error"),
-    }
+    server.receive(buf)?;
+    // TODO: Assert that we dropped the packet.
+    // FIXME: https://github.com/microsoft/demikernel/issues/1065
+    Ok(())
```

Comment:
I fix this in https://github.com/microsoft/demikernel/pull/1074.

Reply:
Great!